src/draw_functions.py

```diff
@@ -67,48 +67,43 @@ def wave(sample, cr, width, height, colors):
         cr.rel_curve_to(width / (ls - 1) * 0.5, 0.0, \
            width / (ls - 1) * 0.5, height_diff * height, \
            width / (ls - 1), height_diff * height)
-    cr.line_to(width, height)
-    cr.line_to(0, height)
-    cr.fill()
-
-def line(sample, cr, width, height, colors, thickness):
-    set_source(cr, height, colors)
-    ls = len(sample)
-    cr.move_to(0, (1.0 - sample[0]) * height)
-    cr.set_line_width(thickness)
-    for i in range(ls - 1):
-        height_diff = (sample[i] - sample[i+1])
-        cr.rel_curve_to(width / (ls - 1) * 0.5, 0.0, \
-           width / (ls - 1) * 0.5, height_diff * height, \
-           width / (ls - 1), height_diff * height)
-    cr.stroke()
+    if fill == False:
```

Comment:
Please use `not var` and `var` instead of `var == False` and `var == True` respectively (not only here, everywhere)

Reply:
Oops, sorry, I forgot one could do that. I'll go ahead and fix it.